src/IntervalMatrices.jl

```diff
@@ -45,9 +40,9 @@ include("correction_hull.jl")
 export AbstractIntervalMatrix,
        IntervalMatrix
 
-export opnorm,
-       left,
-       right,
```

Comment:
I dunno, also, having new names instead of `inf` and `sup` may be confusing. Maybe we just should call these `inf` and `sup` since these are used in `Interval` and `IntervalBox`.

Reply:
:+1: for `import`ing and reusing `inf` and `sup`.